test/fw/ptl/utils/plugins/ptl_test_data.py

```diff
@@ -204,76 +209,13 @@ def __save_home(self, test, status, err=None):
-            output_tar = snap_out.split(":")[1]
-
-        snap_out_dest = os.path.join(datadir, os.path.basename(snap_out))
-
-        snap_out_r = output_tar.strip(" ")
-        ret = self.du.run_copy(current_host, snap_out_r, snap_out_dest,
-                               level=logging.DEBUG2)
-        if ret['rc'] != 0:
-            _msg = 'Failed to copy generated diag from'
-            _msg += ' %s to %s' % (snap_out_r, snap_out_dest)
-            f.write(_msg + '\n')
-            self.logger.error(_msg)
-            f.close()
-            return
-        else:
-            self.du.rm(server_host, path=snap_out, force=True,
-                       level=logging.DEBUG2)
-        snap_out_dest = snap_out_dest.rstrip('.tgz')
-        cmd = 'tar -xf '
-        cmd += snap_out_dest + '.tgz'
-        cmd += ' -C ' + datadir
-
-        ret = self.du.run_cmd(current_host, cmd, level=logging.DEBUG2)
-
-        self.du.rm(current_host, path=snap_out_dest + '.tgz',
-                   force=True, level=logging.DEBUG2)
-
-        if ret['rc'] != 0:
-            _msg = 'Failed to untar %s.tgz' % snap_out_dest
-            _msg += '\n\n' + '\n'.join(ret['err']) + '\n\n'
-            f.write(_msg + '\n')
-            self.logger.error(_msg)
-            f.close()
-            return
-        if len(add_hosts) > 0:
-            for host in add_hosts:
-                dest = os.path.join(snap_out_dest, host + "_snapshot_logs")
-                self.du.mkdir(current_host, path=dest, mode=0o755,
-                              parents=True, logerr=False, level=logging.DEBUG2)
-                sub_snap_log = os.path.join(snap_out_dest,
-                                            host + "_snapshot.tgz")
-                cmd = "tar -xf " + sub_snap_log
-                cmd += " -C " + dest
-                cmd += " --strip-components 1"
-                ret = self.du.run_cmd(current_host, cmd, level=logging.DEBUG2)
-                if ret['rc'] != 0:
-                    _msg = 'Failed to extxract tar %s' % sub_snap_log
-                    _msg += '\n\n' + '\n'.join(ret['err']) + '\n\n'
-                    f.write(_msg + '\n')
-                    self.logger.error(_msg)
-                    f.close()
-                    return
-                self.du.rm(current_host, path=sub_snap_log,
-                           force=True, level=logging.DEBUG2)
-
-        dest = 'PBS_' + current_host.split('.')[0] + '.tar.gz'
-        os.chdir(datadir)
-        cmd = "tar -zcf " + dest + " "
-        cmd += os.path.basename(snap_out).rstrip('.tar.gz')
-
-        ret = self.du.run_cmd(current_host, cmd, level=logging.DEBUG2)
-        if ret['rc'] != 0:
-            _msg = 'Failed to tar %s' % snap_out_dest
-            _msg += '\n\n' + '\n'.join(ret['err']) + '\n\n'
-            f.write(_msg + '\n')
-            self.logger.error(_msg)
-            f.close()
-            return
+        snap_out = ret['out'][0]
+        snap_out_dest = (snap_out.split(":")[1]).strip(" ")
```

Comment:
you don't need to provide " " to strip, it does that by default, so you can call it simply as 'strip()'